pipeline/dbt/models/intermediate/sources/carif_oref/int_carif_oref__structures.sql

```diff
@@ -0,0 +1,66 @@
+WITH organismes_formateurs AS (
+    SELECT * FROM {{ ref('stg_carif_oref__organismes_formateurs') }}
+),
+
+organismes_formateurs__contacts AS (
+    SELECT * FROM {{ ref('stg_carif_oref__organismes_formateurs__contacts') }}
+),
+
+coordonnees AS (
+    SELECT * FROM {{ ref('stg_carif_oref__coordonnees') }}
+),
+
+actions AS (
+    SELECT * FROM {{ ref('stg_carif_oref__actions') }}
+),
+
+formations AS (
+    SELECT * FROM {{ ref('stg_carif_oref__formations') }}
+),
+
+date_maj AS (
+    SELECT
+        actions.numero_organisme_formateur,
+        MAX(COALESCE(actions.date_maj, formations.date_maj)) AS date_maj
+    FROM actions
+    LEFT JOIN formations
+        ON actions.numero_formation = formations.numero
+    GROUP BY actions.numero_organisme_formateur
+),
+
+final AS (
+    -- sort and distinct by numero to keep the first action with the same numero
+    -- because there are several contacts available for the same organismes_formateurs.numero
+    SELECT DISTINCT ON (organismes_formateurs.numero)
+        organismes_formateurs.numero                              AS "id",
+        coordonnees.hash_adresse                                  AS "adresse_id",
+        organismes_formateurs.siret_formateur__siret              AS "siret",
```

Comment:
peut-etre que staging est en retard d'une itération mais actuellement, tous les SIRET sont vides dans cette table.

Reply:
bien vu, merci ! + ajout d'un test manquant `not_null` sur le modele staging associé